app/src/main/res/drawable/ic_launcher_donor_benefit_foreground.xml

```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This icon looks _very_ different from our standard "W" launcher icon. (Not just the "heart" shape, but the size and font of the "W" is different from the default.)
Where did it come from, and how was it generated?
Ideally the icon should be the same "W" as the default, with the "heart" in the same spot as the "star" shape in our Beta/Alpha/Dev icons.

Reply:
I exported this from the Figma and used the Android Resource manager to import this. I have requested design for a new app icon similar to the alpha icons.